plugins/modules/vault_pki_generate_certificate.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2022, Florent David <florent.david@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+  module: vault_pki_generate_certificate
+  version_added: 2.3.0
+  author:
+    - Florent David <florent.david@gmail.com>
```

Comment:
I believe we're removing/discouraging email addresses now in author fields and in copyright line too, you can replace email with your github ID in parentheses
```suggestion
    - Florent David (@Ripolin)
```

Reply:
Fixed in commit d7c4dc98df2ae95e50ad4da21d3be935793f6668